admin/api_access_tracking.go

```diff
@@ -132,7 +132,7 @@ func (r ListAccessLogsByClusterNameApiRequest) Start(start int64) ListAccessLogs
 }
 
 func (r ListAccessLogsByClusterNameApiRequest) Execute() (*MongoDBAccessLogsList, *http.Response, error) {
-	return r.ApiService.listAccessLogsByClusterNameExecute(r)
```

Comment:
@fmenezes To be able to use mocks along with the implementation our execute method. using interface to call ListAccessLogsByClusterNameExecute() (which can be replaced by mock.

Any form of casting to implementation will also prevent mocking here. 
We inherited this API design from upstream.